.github/workflows/main.yml

```
@@ -1 +1,52 @@
+name: CI
 
+on: [pull_request]
+
+jobs:
+  phpstan:
+    runs-on: ubuntu-latest
```

J'ai tendance à utiliser une version fixe de l’image et la mettre à jour à la main manuellement. Ça a tendance à péter lorsque `ubuntu-latest` change de version, et généralement c'est pas le bon moment pour corriger la CI.

```suggestion
    runs-on: ubuntu-24.04
```